README.md

```diff
@@ -3,21 +3,34 @@
 
 [![Visual Studio Marketplace](https://vsmarketplacebadge.apphb.com/version/JFrog.jfrog-vscode-extension.svg)](https://marketplace.visualstudio.com/items?itemName=JFrog.jfrog-vscode-extension)
 
+# Table of Contents
+
+- [General](#general)
+- [Viewing Dependencies Information](#viewing-dependencies-information)
+- [Configuring JFrog Xray](#configuring-jfrog-xray)
+  - [Proxy configuration](#proxy-configuration)
+    - [Proxy Authorization](#proxy-authorization)
+- [Npm Projects](#npm-projects)
+- [Go Projects](#go-projects)
+- [Pypi Projects](#pypi-projects)
+- [Troubleshooting](#troubleshooting)
+- [License](#license)
+- [Building and Testing the Sources](#building-and-testing-the-sources)
+- [Code contributions](#code-contributions)
+  - [Guidelines](#guidelines)
+
 ## General
 
 The cost of remediating a vulnerability is akin to the cost of fixing a bug.
 The earlier you remediate a vulnerability in the release cycle, the lower the cost.
 JFrog Xray is instrumental in flagging components when vulnerabilities are discovered in production systems at runtime,
 or even sooner, during the development.
 
-The JFrog VS Code Extension adds JFrog Xray scanning of npm project dependencies to your VS Code IDE.
+The JFrog VS Code Extension adds JFrog Xray scanning of npm, Go and Python project dependencies to your VS Code IDE.
```

Comment:
JFrog Xray scanning of npm, Go and Python -->  JFrog Xray scanning of **npm**, **G**o and **Python**
(I highlighted the package managers names).